governance/parameter-docs/param-proportional-kick-incentive.md

```diff
@@ -9,24 +9,24 @@ Technical Docs:
 ```
 
 ## Description
-* What is this parameter?
-* How does it fit in with the protocol and surrounding contracts?
+The Proportional Kick Incentive parameter controls the percentage of the target amount of DAI to raise from bidders (debt + stability fees + liquidation penalty) that will be destined to incentivize keepers when liquidating a vault or resetting an already existing auction.
 
 ## Purpose
-* Why does this parameter exist?
+Governance may periodically increase the Debt Floor. When this happens, it is usually because gas has become so expensive that it impacts the efficiency of liquidations — that is, the cost of calling auction-related functions may exceed the value of the collateral to be liquidated. Incentives may be used as a remedy to this potential issue and possibly as a way for the protocol to keep the Debt Floor lower. The Proportional Kick Incentive ensures that auction initialization or reset is being compensated to cover keepers gas costs and spurs the prompt liquidation of unsafe vaults.
```

Comment:
"Governance may periodically increase the Debt Floor." - Don't start with this, it's a bit of a non-sequitur. 'The Proportional Kick Incentive exists to ensure that auctions are triggered despite the cost of gas on the Ethereum blockchain' Something like this?

Reply:
Bring up Debt Floor later, but it isn't the primary purpose of this. Also, the flat kick incentive is way better suited for dealing with stuff near the Debt Floor. The main focus here is making sure large liquidations happen promptly in high-gas conditions.